demisto_sdk/commands/validate/tests/validators_test.py

```diff
@@ -678,7 +678,7 @@ def test_create_ignored_errors_list(self):
                              "LI101"]
         error_list = validate_manager.create_ignored_errors_list(errors_to_exclude)
         assert ["BA101", "BA102", "BA103", "BA104", "BA105", "BA106", "BA107", "BA108", "BA109",
-                "BA110", 'BA111', "BA112", "BA113", "BA114", "BA115", "BC101", "BC102", "BC103", "BC104"] <= error_list
+                "BA110", 'BA111', "BA112", "BA113", "BA114", "BA115", "BA116", "BC101", "BC102", "BC103", "BC104"] <= error_list
```

Comment:
odd behavior here, please consult with @DeanArbel whether to fix this `<=` or just add `BA116` like you did here and leave this window broken.